lib/http1.c

```diff
@@ -1337,7 +1337,7 @@ void h2o_http1_accept(h2o_accept_ctx_t *ctx, h2o_socket_t *sock, struct timeval
     sock->data = conn;
     h2o_linklist_insert(&ctx->ctx->http1._conns, &conn->_conns);
 
-    H2O_PROBE_CONN(H1_ACCEPT, &conn->super, conn->sock, &conn->super);
+    H2O_PROBE_CONN(H1_ACCEPT, &conn->super, conn->sock, &conn->super, h2o_conn_get_uuid(&conn->super));
```

Comment:
That's what `H2O_PROBE*()` does: https://github.com/h2o/h2o/blob/909da44bf32d7da1ad64cea871badabcffda34f5/lib/probes_.h#L48-L54

> BTW, do we need to pass each member of conn when we also pass h2o_conn_t * as a pointer? I'm not requesting a change, though I thought that you created a mechanism for extracting multiple members of a struct. Hence asking.

I think no. For example `uint64_t conn_id` in `h2o:h3s_accept` probe, h2olog doesn't need it. It can be extracted from `h2o_conn_t *conn`. However, it might be convenient for other dtrace tools, although I'm not sure the use-case when h2olog gets production-ready.

Reply:
> That's what `H2O_PROBE*()` does: 

Point taken. Yeah it looks like a function call, but the arguments are actually evaluated lazily, because it is a macro.

>  However, it might be convenient for other dtrace tools, although I'm not sure the use-case when h2olog gets production-ready.

Yeah, any dtrace tools can read values given a pointer. We already rely on that for things like `quicly_rtt_t`. That said, I'm happy to leave this issue to another PR (of using multiple arguments for passing members of `h2o_conn_t` while passing `h2o_conn_t` as a pointer), if that's your preference.

Reply:
Agreed. I'm willing to tackle with that redundancy 😄